_template.ebuild

```diff
@@ -24,6 +24,7 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="test"
+GO111MODULE=auto
```

Comment:
@mohamednizar this line is not needed, I'll update ebuild later (it should use `go-mod` eclass instead of `golang-vcs-snapshot` to support modules)

Reply:
The line is removed